common/plugins/ipam.go

```diff
@@ -39,7 +39,7 @@ type Route struct {
 	Nexthop *net.IP
 }
 
-// Sort an array of routes by mask, narrow first
+// ByMask sort an array of routes by mask, narrow first.
```

Comment:
Don't see any difference in your comment with the actual diff here. What am I missing?